opm/material/common/UniformXTabulated2DFunction.hpp

```diff
@@ -317,6 +317,16 @@ class UniformXTabulated2DFunction
     template <class Evaluation>
     Evaluation eval(const Evaluation& x, const Evaluation& y, bool extrapolate=false) const
     {
+        Evaluation alpha,beta1,beta2;
+        unsigned i,j1,j2;
+        findPoints(i, j1, j2, alpha, beta1,beta2, x, y, extrapolate);
+        return eval(i, j1, j2, alpha, beta1, beta2,x, y, extrapolate);
+    }
+
+        template <class Evaluation>
```

Comment:
Nitpicks: indentation here and line 328. Space around `=`.

Not a nitpick: Avoid default parameters for new functions.

Reply:
think it is done